freqtrade/resolvers/strategy_resolver.py

```diff
@@ -66,13 +80,24 @@ def __init__(self, config: Optional[Dict] = None) -> None:
 
         self._strategy_sanity_validations()
 
-    def _override_attribute_helper(self, config, attribute: str):
+    def _override_attribute_helper(self, config, attribute: str, default):
```

Comment:
should `default` be `None` by default?

Reply:
there is not really a point tbh - it's always passed in within the loop and it's not used anywhere else ...